code/ui/components/dropdown/DropdownOption.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+
+using Sandbox.UI;
+using Sandbox.UI.Construct;
+
+namespace TTTReborn.UI
+{
+    public partial class DropdownOption : TTTPanel
+    {
+        public Dropdown Dropdown { get; set; }
+
+        public Label TextLabel { get; private set; }
+
+        public Action<TTTPanel> OnSelect { get; set; }
+
+        public DropdownOption(Panel parent = null, string text = null) : base(parent)
```

Comment:
Can we just do empty string as a default instead of null for `text`?

Reply:
And what's the benefit? <https://github.com/TTTReborn/tttreborn/pull/177/files#diff-c74c736f039bae78fc73d7b0f9b420ba7a81707b4c27469924b1d550e428dde2R20>

Reply:
It's generally better to use String.Empty as it's just safer overall since you won't get null refs.
It's just better practice as far as I'm aware.

Reply:
^ yeah just better practice basically, if you have the option to use something empty, you should.